tests/fft_test.py

```diff
@@ -288,18 +286,17 @@ def testFftErrors(self, inverse, real, hermitian):
     self.assertRaises(
         ValueError, lambda: func(rng([2, 3], dtype=np.float64), axis=[-3]))
 
-  @parameterized.named_parameters(jtu.cases_from_list(
-      {"testcase_name": "_inverse={}_real={}_shape={}_axes={}_norm={}".format(
-          inverse, real, jtu.format_shape_dtype_string(shape, dtype), axes, norm),
-       "axes": axes, "shape": shape, "dtype": dtype, "inverse": inverse, "real": real, "norm":norm}
-      for inverse in [False, True]
-      for real in [False, True]
-      for dtype in (real_dtypes if real and not inverse else all_dtypes)
-      for shape in [(16, 8, 4, 8), (16, 8, 4, 8, 4)]
-      for axes in [(-2, -1), (0, 1), (1, 3), (-1, 2)]
-      for norm in FFT_NORMS
-      ))
-  def testFft2(self, inverse, real, shape, dtype, axes, norm):
+  @jtu.sample_product(
+    [dict(inverse=inverse, real=real, dtype=dtype)
+     for inverse in [False, True]
+     for real in [False, True]
+     for dtype in (real_dtypes if real and not inverse else all_dtypes)
+    ],
+    shape=[(16, 8, 4, 8), (16, 8, 4, 8, 4)],
+    axes=[(-2, -1), (0, 1), (1, 3), (-1, 2)],
+    norm=FFT_NORMS,
+  )
+  def testFft2_(self, inverse, real, shape, dtype, axes, norm):
```

Comment:
Why the trailing underscore?

Reply:
Because the test case names are formed by suffixing numbers, so there's a clash between `testFft`'s second test case and `testFft2`'s first one.